_posts/2019-03-15-jan-feb-rubygems-update.md

```diff
@@ -0,0 +1,28 @@
+---
+title: January and February 2019 RubyGems Updates
+layout: post
+author: Stephanie Morillo
+---
+
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in January and February.
+
+## rubygems.org news
+
+In last two months, we released a new search API powered by Elasticsearch, which resolves multiple issues including [missing search results](https://github.com/rubygems/rubygems.org/issues/972) and [slow performance](https://github.com/rubygems/rubygems.org/issues/1256). Thanks to @lucianosousa, we are now using Rails 5.2.2. We would also like to let you know that we sprinkled some styling to our email templates and now they are no longer being marked as spam by some email providers. We haven't received any new help ticket for email being lost in last month—a good sign!
```

Comment:
> released a new search API

we didn't release new API. we updatied existing one to use Elasticsearch instead of Postgres ILIKE.

Reply:
@sonalkr132 updated